### DISCUSSION TITLE
DSO500K-U Firmware

### DISCUSSION THREAD

**Discussion by drawgas** [fhdm-dev/scoppy#140] [Q&A] [answered]
As DSO500K-U got discontinued - can you use DSO-500K firmware for DSO500K-U?

**Comment by fhdm-dev** [comment, marked as answer]
No, but the [latest firmware](https://oscilloscope.fhdm.xyz/wiki/firmware-versions) is now also available for the DSO500K-U :smile:

**Reply by drawgas** [reply]
Thank you for that :)
